x-pack/plugins/entity_manager/server/lib/entities/find_entity_definition.ts

```diff
@@ -144,33 +143,14 @@ async function getIngestPipelineState({
     .filter(({ type }) => type === 'ingest_pipeline')
     .map(({ id }) => id);
 
-  const [ingestPipelines, ingestPipelinesStats] = await Promise.all([
-    esClient.ingest.getPipeline({ id: ingestPipelineIds.join(',') }, { ignore: [404] }),
-    esClient.nodes.stats({
```

Comment:
`node_stats` requires privileges that are not included in the api key. we used to always call this api with system user so we never faced privilege issues but with this change it's no longer the case. I'm simply removing the ingest pipelines stats since it has low value for now and will have even less in the future.

Reply:
I'm fine with dropping it for now, as we come back to building a UI to manage these things and the future materialisation settings we may want to add new data collection for that and I believe that would be the right time to add a dedicated role to capture those privileges for the user who is able to enable materialisation.